frontend/src/components/Employee/Orders.tsx

```diff
@@ -0,0 +1,219 @@
+import SendIcon from '@mui/icons-material/Send';
```

Comment:
We shouldn't have this file anymore after my most recent changes. Is there a reason for it being included?

Reply:
not sure I still see it in my order.tsx